pyproject.toml

```diff
@@ -16,7 +16,8 @@ keywords = ["cython", "setuptools"]
 authors = [{name = "Luper Rouch", email= "luper.rouch@gmail.com"}]
 maintainers = [{ name = "Marco Treglia", email = "mtreglia@gopro.com" }]
```

Comment:
I suggest you to add you as maintainers ;)

Reply:
I will do it